LabExT/Tests/Measurements/InsertionLossSweep_test.py

```diff
@@ -32,7 +33,8 @@ def check_ILsweep_data_output(test_inst, data_dict, params_dict):
                                params_dict['wavelength stop'].value))
 
 
-class Test_InsertionLossSweep(unittest.TestCase):
+@mark_as_laboratory_test
```

Comment:
This test can be run with the supplied simulator classes and does NOT need any instruments attached. No need to classify it as a lab test.

```suggestion
```

Reply:
ups, I overlooked that. Now they are included on CI